lib/mastodon/cli/maintenance.rb

```diff
@@ -254,7 +254,7 @@ def deduplicate_users!
         users = User.where(id: row['ids'].split(',')).sort_by(&:updated_at).reverse
         ref_user = users.shift
         say "Multiple users registered with e-mail address #{ref_user.email}.", :yellow
-        say "e-mail will be disabled for the following accounts: #{user.map { |user| user.account.acct }.join(', ')}", :yellow
+        say "e-mail will be disabled for the following accounts: #{users.map { |user| user.account.acct }.join(', ')}", :yellow
```

Comment:
This is a bug-fix of a pre-existing issue ... should probably merge this in somewhere regardless of what happens on this PR. Let me know if you'd like JUST this change pulled out.

Reply:
Oops, yeah, I'd like that as a separate PR!

Reply:
https://github.com/mastodon/mastodon/pull/28363